toolkit/mozapps/extensions/nsBlocklistService.js

```diff
@@ -515,6 +537,9 @@ Blocklist.prototype = {
     if (!gBlocklistEnabled)
       return;
 
+    if (!gBlocklistFromXML)
+      return;
```

Comment:
We should still download the XML if the Cert relies on it (cf question about keeping a pref per blocklist)